src/utils/articles.js

```diff
@@ -352,4 +352,20 @@ export const twentyTwo = [
     date: new Date('Jan 8 2023, 04:02:00'),
     entry_id: 15,
   },
+  {
+    title: "This is not aspire to inspire",
+    previewText: "I’ve always wanted to do the 100 Days Of Code challenge for a long time, because I see it as an opportunity to stay consistent at improving my programming skills but for one reason or another, I always default on my goals.",
+    author: "Damilare Ololade",
+    readMore: "https://www.linkedin.com/posts/damilareololade_20daylinkedinchallengewithhaoma-tracklearnings-activity-7019491551206649856-gwcK?utm_source=share&utm_medium=member_desktop",
```

Comment:
...and please, can we stick with articles that were actually published on other platforms like medium, DEV.to, Hashnode, Personal blogs, and not LinkedIn posts?

Reply:
An example is this

https://estherolowoloba.medium.com/2022-in-retrospect-e20bef4e5ccd